docs/fsharp/using-fsharp-on-azure/file-storage.md

```diff
@@ -16,11 +16,32 @@ For a conceptual overview of file storage, see [the .NET guide for file storage]
 To use this guide, you must first [create an Azure storage account](/azure/storage/storage-create-storage-account).
 You'll also need your storage access key for this account.
 
-## Create an F# Script and Start F# Interactive
+## Create an F\# script and start F\# interactive
 
 The samples in this article can be used in either an F# application or an F# script. To create an F# script, create a file with the `.fsx` extension, for example `files.fsx`, in your F# development environment.
 
-Next, use a [package manager](package-management.md) such as [Paket](https://fsprojects.github.io/Paket/) or [NuGet](https://www.nuget.org/) to install the `WindowsAzure.Storage` package and reference `WindowsAzure.Storage.dll` in your script using a `#r` directive.
+### How to execute scripts
+
+F# Interactive, `dotnet fsi`, can be launched interactively, or it can be launched from the command line to run a script. The command-line syntax is
+
+```.NET CLI
+> dotnet fsi [options] [ script-file [arguments] ]
+```
+
+### Add packages in a script
+
+Use `#r` `nuget:package name` to install the `Azure.Storage.Blobs` and `Azure.Storage.Sas` and `Azure.Storage.Files` packages and `open` namespaces. Such as
+
+```fsharp
+> #r "nuget: Azure.Storage.Blobs"
+> #r "nuget: Azure.Storage.Sas"
```

Comment:
There is no package called `Azure.Storage.Sas`.  You might be looking for `Azure.Storage.Common`?  I'd strongly recommend we run these from an F# script to make sure there are no more issues like this.